src/web.cpp

```diff
@@ -94,7 +96,7 @@ int crashCount = 0;
 // For Server Sent Events (SSE) support
 // Just reloading page causes register on new channel.  So we need a reasonable number
 // to accommodate "extra" until old one is detected as disconnected.
-#define SSE_MAX_CHANNELS 8
+#define SSE_MAX_CHANNELS 4
```

Comment:
I tried moving this to the iram heap as well, but it cratered.  not sure why, so I just reduced to 4.

Reply:
I suspect because it has C++ classes within the struct.  And those have memory malloc/free behind the scenes.

sizeof(SSESubscription) is 216 bytes (according to VScode, and with my logViewer bool), so array of 4 is 864 bytes.  I think limiting to 4 is okay.